From fac439f93d1afdaed4514472d494841d50efcace Mon Sep 17 00:00:00 2001
From: Konrad Weihmann <kweihmann@outlook.com>
Date: Mon, 22 Aug 2022 17:13:30 +0000
Subject: [PATCH 1/2] MANIFEST: add CHANGES.txt

otherwise when running pure setup.py installation fails
because the file is not part of the pypi release

Signed-off-by: Konrad Weihmann <kweihmann@outlook.com>
Upstream-Status: Submitted [https://github.com/nexB/pkginfo2/pull/2]
---
 MANIFEST.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MANIFEST.in b/MANIFEST.in
index aca2c6c..6788bfc 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1 +1,2 @@
 graft docs/examples/
+CHANGES.txt
-- 
2.34.1

